applications/luci-app-tailscale/Makefile

```diff
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: GPL-3.0-only
+#
+# Copyright (C) 2024 asvow
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=LuCI for Tailscale
+LUCI_DEPENDS:=+tailscale
```

Comment:
For completeness, I've discovered that at least `ip-tiny` is required. So add the dep `ip`. The default busybox command doesn't yet understand `-j`. And tailscale is huge anyway, so a new ip binary is a fraction of that cost.